internal/config/loader.go

```diff
@@ -40,6 +40,7 @@ func LoadConfig(useRegistry bool, profile string, confDir string) (*common.Confi
 		useRegistry, profile, confDir)
 
 	configuration, err := loadConfigFromFile(profile, confDir)
+	fmt.Fprintf(os.Stdout, "config: %v\n", configuration)
```

Comment:
I think this line should be removed

Reply:
sorry, you are right
that was written for my personal test...
forgot

Reply:
@jduranf thanks, I've removed this line